_data/blog_posts.yml

```diff
@@ -1,3 +1,12 @@
+- title: "A talk on the new AWS Application Load Balancer, updates to ECS, and Kinesis Analytics"
+  url: "https://blog.gruntwork.io/a-talk-on-the-new-aws-application-load-balancer-updates-to-ecs-and-kinesis-analytics-abb599cb3cb8#.vhbek5ce2"
+  thumbnail_url: "https://i.imgur.com/KkniqPD.png"
```

Comment:
Well, if you add an image into the Medium post itself, it will a) use it as a thumbnail on the [blog.gruntwork](https://blog.gruntwork.io/) page, which will look much better and b) you'll be able to use that image's URL here.